lgc/interface/lgc/PassManager.h

```diff
@@ -51,8 +51,14 @@ class PassManager : public llvm::ModulePassManager {
 public:
   static PassManager *Create();
   virtual ~PassManager() {}
+  template <typename PassBuilderT> bool registerFunctionAnalysis(PassBuilderT &&PassBuilder) {
+    return functionAnalysisManager.registerPass(PassBuilder);
```

Comment:
Why is PassBuilder taken by `&&` but not forwarded? I think this should either take it by `&` or use `std::forward`